src/components/forms/validator.ts

```diff
@@ -89,11 +86,15 @@ export const minMaxLength = (minLen: number, maxLen: number) => (value: string):
 
 export const ADDRESS_REPEATED_ERROR = 'Address already introduced'
 
-export const uniqueAddress = (addresses: string[] | List<string>): GenericValidatorType =>
+export const uniqueAddress = (addresses: string[]): GenericValidatorType =>
   memoize(
-    (value: string): ValidatorReturnType => {
-      const addressAlreadyExists = addresses.some((address) => sameAddress(value, address))
-      return addressAlreadyExists ? ADDRESS_REPEATED_ERROR : undefined
+    (): ValidatorReturnType => {
+      const repeatedAddresses = addresses.filter((element, position, array) => array.indexOf(element) !== position)
```

Comment:
not completely sure, but wouldn't it be more direct to build a `Set`->`Array` and then compare lengths?

Also, I think that `memoize` is no longer needed with this approach.

Reply:
it's done

Reply:
What was wrong with the previous version of the function? I can't find the bug but the description says we had something to fix there.

Reply:
The old `uniqueAddress` received a value to check if that value if inside the array but the function that was calling it called it multiple times a lot of times with the same value after 10 elements are inside the array, so I found that just checking the array instead of receiving a value fixes it